<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>169</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_accept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_lattice">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>18</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_workspace">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>49</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_workspace">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>137</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diamond Lattice Path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lattice">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>19</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Workspace:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>731</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>81</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vhdl Path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vhdl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>81</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_vhdl">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lineEdit_workspace</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_lattice</zorder>
   <zorder>label</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>pushButton_lattice</zorder>
   <zorder>pushButton_workspace</zorder>
   <zorder>textBrowser</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_vhdl</zorder>
   <zorder>pushButton_vhdl</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_lattice</tabstop>
  <tabstop>pushButton_lattice</tabstop>
  <tabstop>lineEdit_workspace</tabstop>
  <tabstop>pushButton_workspace</tabstop>
  <tabstop>pushButton_accept</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
